Bottom: 77e2dee8725200aeb308fc1b1df47db89ac234d0
Top:    824823bad26806cf6ad46f793fa661ca295a703f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 01:33:47 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index d387f43..0b03204 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
